website/src/pages/use-cases/incident-response.tsx

```diff
@@ -0,0 +1,368 @@
+        quote: `Isn't it nice when you can just run a report and say, 'Here it is' or 'Here it isn't?' Much better than having to say, 'Well, boss, I think we got it all.'`,
+        by: 'Jon Kohler, Technical Director of Solution Engineering, Nutanix',
+        logoImage: '/external-logos/nutanix-logo.svg',
+        linkText: 'Read the case study',
+        link: '/case-studies/nutanix-fixed-log4j-with-sourcegraph',
+        logoAlt: 'Nutanix Logo',
+    },
+]
+
+const blogListItems = [
+    {
+        title: 'Log4j Log4Shell 0-day: find, fix, and track affected code',
+        description:
+            'In the biggest security vulnerability incident since Heartbleed, Sourcegraph co-founder and CEO Quinn Slack shared how you can find affected code, automate fixes, and track progress.',
+        type: 'Blog post',
+        image: 'https://sourcegraphstatic.com/blog/log4j/log4j-blog-thumbnail.png',
+        href: '/blog/log4j-log4shell-0-day',
+    },
+    {
+        title: `The real weakest link in software supply chain security (it's not open source)`,
+        description:
+            'Using open source code can jump-start development but it can also expose you to security vulnerabilities. In this post, learn how to design an effective vulnerability management process that can make dependencies visible and mitigation less time-consuming.',
+        type: 'Blog post',
+        image: 'https://storage.googleapis.com/sourcegraph-assets/blog/third-party-open-source-vulnerabilities.png',
+        href: '/blog/real-weakest-link-in-software-supply-chain-security',
+    },
+    {
+        title: 'How to remove secrets from your codebase',
+        description:
+            'Back in early 2021, Sourcegraph stored infrastructure and service passwords in private repositories. Security engineer André Eleuterio moved every secret to a secure vault and used code search to ensure the move was successful and complete.',
+        type: 'Blog post',
+        image: 'https://sourcegraphstatic.com/blog/securing-sourcegraph-eliminating-secrets.png',
+        href: '/blog/eliminate-secrets-from-codebase-with-universal-code-search',
+    },
+]
+
+const UseCasePage: FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Incident Response | Sourcegraph',
+            description:
+                'Identify the root cause of an incident, understand its potential impact, fix the issue everywhere in your codebase. Incident response from Sourcegraph.',
+            image: 'https://about.sourcegraph.com/sourcegraph-og.png',
+        }}
+        className="use-cases-page"
+        heroAndHeaderClassName={`${styles.useCaseHeader} navbar-light`}
+        hero={
+            <>
+                <div className="bg" />
+                <div className="container pb-4">
+                    <div className="row">
+                        <div className="col-lg-7 mb-8 mt-7">
```

Comment:
To keep our new Use Cases pages consistent, Brett [brought up](https://github.com/sourcegraph/about/pull/5271#discussion_r857943812) in my PR to have some more progressive spacing for tablet sized screens. So the buttons don't take up the whole screen width on `md` screens. Can you change `col-md-7` to align with this? We can just make a cleanup story to ensure these pages are unified after they're merged too!

![image](https://user-images.githubusercontent.com/59381432/165857168-4785a0ed-89a2-415f-a2f6-62583603f201.png)
TO =>
![image](https://user-images.githubusercontent.com/59381432/165857255-15adac3a-4754-4091-8583-4c257b3f02e1.png)

Reply:
Sure thing! This was quick, so I added it here: f76e7e7.